certified-connectors/PDFcross/apiDefinition.swagger.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "PDFcross is a service for manipulating PDF files, such as merging PDF files into one file.",
+    "version": "1.0.0",
+    "title": "PDFcross",
+    "contact": {
+      "name": "PotCross Support",
+      "url": "https://pot-cross.com/en/",
+      "email": "support@pot-cross.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://pdfcross.com/en"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://pdfcross.com/en/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files;Productivity"
+    }
+  ],
+  "host": "pdfcross-dev.an.r.appspot.com",
```

Comment:
@nabe-hiromichi Please note that we do not certify a connector that has a dev or test or stage host URL. Please confirm that the host URL is the production one?

Reply:
I fix the host URL.